irohad/ametsuchi/impl/postgres_wsv_query.cpp

```diff
@@ -76,263 +39,288 @@ namespace iroha {
           factory_(factory),
           log_(logger::log("PostgresWsvQuery")) {}
 
+    template <typename T>
+    boost::optional<std::shared_ptr<T>> PostgresWsvQuery::fromResult(
```

Comment:
Maybe move to types?

Reply:
It will be cumbersome to log the error then.